backend/src/core/middlewares/__init__.py

```diff
@@ -16,3 +17,4 @@ def add_middlewares(app: FastAPI) -> None:
     app.add_middleware(CORSMiddleware, **add_cors_middleware(app))
     app.add_middleware(BaseHTTPMiddleware, dispatch=logging_middleware)
     app.add_middleware(CorrelationIdMiddleware)
+    app.add_middleware(SessionMiddleware, secret_key="My secret key")
```

Comment:
What is `secret_key` variable? If it's sensitive, can you move it to the ENV?

Reply:
Removed `SessionMiddleware` that is not used right now